salt/metalk8s/kubeadm/init/addons/kube-proxy.sls

```diff
@@ -0,0 +1,219 @@
+{%- from "metalk8s/map.jinja" import networks with context %}
+
+{%- set image = "localhost:5000/metalk8s-2.0/kube-proxy:1.11.7" -%}
+
+{% set kubeconfig = "/etc/kubernetes/admin.conf" %}
+{% set context = "kubernetes-admin@kubernetes" %}
```

Comment:
Do we want to set these somewhere in the master config? It seems the `kubernetes` module supports having these values in either master/minion config or pillar values.

Reply:
So, here's the deal: currently, we run all these states on the `bootstrap` minion, which may be fine, but is not OK once we grow the cluster and such. The rationale is simple: this state (and others) should run just once.

This relates to #626 which would allow us to run this state only once, on the Salt master. In order for that to work, the master indeed needs creds, and ideally those are referred to from its configuration file, which we can do.

As to which creds to use: I'd like to create a `ClusterRole` and a `Role` that grants whatever access the Salt master requires to do its job (we can create those using the `admin` creds), then create a cert for the Salt master with bindings to said `(Cluster)Role`. That way, we can deploy Salt master(s) without handing out any non-required authorizations.

Reply:
Created #691.